tests/unit/GHDL.py

```diff
@@ -0,0 +1,89 @@
+#   Patrick Lehmann                                                                                                    #
+#                                                                                                                      #
+# License:                                                                                                             #
+# ==================================================================================================================== #
+# Copyright 2017-2021 Patrick Lehmann - Boetzingen, Germany                                                            #
+#                                                                                                                      #
+# Licensed under the Apache License, Version 2.0 (the "License");                                                      #
+# you may not use this file except in compliance with the License.                                                     #
+# You may obtain a copy of the License at                                                                              #
+#                                                                                                                      #
+#   http://www.apache.org/licenses/LICENSE-2.0                                                                         #
+#                                                                                                                      #
+# Unless required by applicable law or agreed to in writing, software                                                  #
+# distributed under the License is distributed on an "AS IS" BASIS,                                                    #
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.                                             #
+# See the License for the specific language governing permissions and                                                  #
+# limitations under the License.                                                                                       #
+#                                                                                                                      #
+# SPDX-License-Identifier: Apache-2.0                                                                                  #
+# ==================================================================================================================== #
+#
+"""Unit tests for executable ``ghdl``."""
+from os                   import environ as os_environ
+from pathlib              import Path
+from pytest               import mark
+from unittest             import TestCase
+
+from pyEDAA.CLITool.GHDL  import GHDL
+from .                    import Helper
+
+
+@mark.xfail      # XXX: workaround for problems in GHA
+class CommonOptions(TestCase, Helper):
+	_binaryDirectoryPath = Path(os_environ["GHDL_PREFIX"]) / "bin"
```

Comment:
I believe this should be `getenv`, defaulting to `/usr/local/`. Moreover, `_binaryDirectoryPath` might be empty if GHDL_PREFIX is unset. E.g., if ghdl is available in the PATH already.

Reply:
Does the GHDL setup action add GHDL in path?

Reply:
Changed in 8046ed1a.